<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TracksPage</class>
 <widget class="QWidget" name="TracksPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1080</width>
    <height>670</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tracks</string>
  </property>
  <widget class="QPushButton" name="manual_add_button">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>170</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Manually Enter Track</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_from_folder_button">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>170</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Files from Folder</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_library_button">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>20</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>false</bold>
     <kerning>true</kerning>
    </font>
   </property>
   <property name="toolTip">
    <string extracomment="Nuke ENTIRE library, ALL music files, ALL image files, &amp; ENTIRE database"/>
   </property>
   <property name="text">
    <string>âš  DELETE ENTIRE LIBRARY</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tracks_table">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>1031</width>
     <height>601</height>
    </rect>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
   </property>
   <property name="alternatingRowColors">
    <bool>true</bool>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
   </property>
   <property name="sortingEnabled">
    <bool>true</bool>
   </property>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>true</bool>
   </attribute>
  </widget>
  <widget class="QPushButton" name="add_files_button">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>170</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Track File(s)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
